CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
   * Bump minimum supported deployment target to iOS 16+
 * BraintreeVenmo
   * Update `BTVenmoRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+  * Remove `fallbacktoWeb` property from the `verifyAppSwitch` method; all Venmo flows will now use universal links to switch to the Venmo app or fallback to the web flow if the Venmo app is not installed
```

Comment:
`verifyAppSwitch` is an internal method, so a merchant isn't going to be concerned with it. Instead, it's more important to them that the `fallbackToWeb` setting was removed from the public `BTVenmoRequest` class

Reply:
Thanks! I'll update